07-regression1.Rmd

```diff
@@ -605,9 +652,9 @@ plot_ly() %>%
 ```
 
 We can see that the predictions in this case, where we have 2 predictors, form
-a surface instead of a line. Because the newly added predictor, number of
-bedrooms, is correlated with price (USD) (meaning as price changes, so does
-number of bedrooms) and not totally determined by house size (our other predictor),
+a surface instead of a line. Because the newly added predictor (number of bedrooms) is 
+related to price (as price changes, so does number of bedrooms)
+and is not totally determined by house size (our other predictor),
 we get additional and useful information for making our
 predictions. For example, in this model we would predict that the cost of a
 house with a size of 2,500 square feet generally increases slightly as the number
```

Comment:
I think we should add an "Additional resources" section here. Also, ISLR does cover KNN regression in Chapter 3.

Reply:
additional resources for regression happens at the end of regression II (just like in the classification chapter pair)